test/Test_Synthesizer.ml

```diff
@@ -180,9 +180,34 @@ let all_mapR_ge_l_0 () =
     constants = []
   } in Alcotest.(check string) "identical" "(all (map-fixR-int-geq l 0))" result.string
 
+let real_add () =
+  let open Synthesizer in
+  let result = solve ~config:{ Config.default with logic = Logic.of_string "LRA" } {
+    arg_names = [ "x" ; "y" ; "z" ];
+    inputs = List.map ~f:(Array.map ~f:(fun r -> Value.Real r))
+                [ [| 1.7 ; 2. ; 3. ; 4.1 ; 5.; 6.|];
+                [| 1. ; 0.5; 1.; 0.25 ; (-0.2); 1. |]; 
+                [| 2.7 ; 2.5; 3.; 4.35 ; 4.8; 8.1 |] ];
+    outputs = Array.map ~f:(fun b -> Value.Bool b)
+                        [| true; true; false; true; true; false |];
+    constants = []
+  } in Alcotest.(check string) "identical" "(= (+ x y) z)" result.string
+
+let real_div () =
+  let open Synthesizer in
+  let result = solve ~config:{ Config.default with logic = Logic.of_string "LRA" } {
+    arg_names = [ "x" ; "y" ];
+    inputs = List.map ~f:(Array.map ~f:(fun r -> Value.Real r))
+                [ [| 3. ; 7. ; -1.5 ; -4.5 ; -3.2 ; 8.1 |]; 
+                [| 6.0 ; 15.6 ; -3.0 ; -9.0 ; -6.0 ; 16.2 |] ];
+    outputs = Array.map ~f:(fun b -> Value.Bool b)
+                        [| true ; false ; true ; true ; false ; true |];
+    constants = []
+  } in Alcotest.(check string) "identical" "(= (/ y 2.) x)" result.string
+
```

Comment:
How easy is it to add one more example with multiplication?

Reply:
I can try again. I actually wrote this with the intention of the output being (= y (* x 2.)), but it output division instead so I just changed the expected result to that. I'm not sure how to make it go to multiplication first and not division since anything can be written either way, but I can play around with it for a little and see.

Reply:
oh yea it's true, it could always pick division instead of multiplication.. try with a different example and see if it would work